docs/admin/telemetry.md

```diff
@@ -22,6 +22,8 @@ Settings section of the platform UI.
 It is also possible to disable in the `flowforge.yml` configuration file. This
 overrides whatever option is set in the Admin Settings UI.
 
+**IMPORTANT: Licensed installations cannot disable telemetry**
```

Comment:
Good first iteration, but I think we need to word this slightly differently as we don't talk about 'installations' generally.

We will need to document the behaviours around licensing in general with the changes coming in the other tasks.  So lets make sure we do a 'big picture' review of the licensing docs as part of this work once we have the full set of behaviours defined.

Reply:
Added to task https://github.com/flowforge/flowforge/issues/1745 to #1638